cas/local.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package cas implements a content-addressable-store on disk.
+// It leverages the `diskv` package to store items in a simple
+// key-value blob store: https://github.com/peterbourgon/diskv
+package cas
+
+import (
+	"fmt"
+	"io"
+	"net/url"
+	"os"
+
+	"github.com/coreos/rocket/pkg/keystore"
+
+	"github.com/coreos/rocket/Godeps/_workspace/src/github.com/appc/spec/aci"
+
+	"github.com/coreos/rocket/Godeps/_workspace/src/golang.org/x/crypto/openpgp"
+)
+
+func NewLocal(aciurl, sigurl string) *Local {
```

Comment:
@jzelinskie would be ecstatic to receive a patch making this consistent everywhere ;-)